src/views/explore/explore.jsx

```diff
@@ -84,14 +93,20 @@ var Explore = injectIntl(React.createClass({
                 break;
             }
         }
-        window.location = window.location.origin + '/explore/' + newType + '/' + this.props.tab;
+        window.location = window.location.origin + '/explore/' + newType + '/' + this.props.tab + '/' + this.props.mode;
+    },
+    changeSortMode: function (name, value) {
+        if (this.props.acceptableModes.indexOf(value) !== -1) {
+            window.location = window.location.origin + '/explore/' +
+            this.props.itemType + '/' + this.props.category + '/' + value;
+        }
     },
     getBubble: function (type) {
         var classes = classNames({
             active: (this.props.category === type)
         });
         return (
-            <a href={'/explore/' + this.props.itemType + '/' + type + '/'}>
+            <a href={'/explore/' + this.props.itemType + '/' + type + '/' + this.props.mode}>
```

Comment:
Ah ok – now I see why the default allows for an empty string. I'm wondering if instead it would work for the three options to then be `['', 'popular', 'recent']` and just have the label for `''` in the dropdown be `'trending'`?